org.virt_manager.virt-viewer.yml

```diff
@@ -0,0 +1,186 @@
+          - "-Dstorage_dir=disabled"
+          - "-Dstorage_fs=disabled"
+          - "-Dstorage_lvm=disabled"
+          - "-Dstorage_iscsi=disabled"
+          - "-Dstorage_scsi=disabled"
+          - "-Dstorage_mpath=disabled"
+          - "-Dstorage_disk=disabled"
+          - "-Dstorage_rbd=disabled"
+          - "-Dstorage_sheepdog=disabled"
+          - "-Dstorage_gluster=disabled"
+          - "-Dstorage_zfs=disabled"
+          - "-Dstorage_vstorage=disabled"
+        build-options:
+          cflags: "-I/app/include/tirpc -ltirpc"
+        sources:
+          - type: archive
+            url: https://libvirt.org/sources/libvirt-8.4.0.tar.xz
+            sha256: 601a6e9bb03a43d05659f63e4a403df253ab0975b4a973f10a5607f3bbc018dd
+          - type: patch
+            path: libvirt-use-monitor-in-xdg-runtime-dir.patch
+  - name: spice-gtk
+    buildsystem: meson
+    build-options:
+      env:
+        PYTHONPATH: /app
+    config-opts:
+      - "-Dvapi=enabled"
+      - "-Dwebdav=enabled"
+      - "-Dgtk_doc=disabled"
+      - "-Dusbredir=disabled"
+      - "-Dpolkit=disabled"
+    sources:
+      - type: archive
+        url: >-
+          https://gitlab.freedesktop.org/spice/spice-gtk/uploads/05c649d612f31b60e4aa44a7c44e611c/spice-gtk-0.41.tar.xz
+        sha256: d8f8b5cbea9184702eeb8cc276a67d72acdb6e36e7c73349fb8445e5bca0969f
+    modules:
+      - name: spice-protocol
+        buildsystem: meson
+        sources:
+          - type: archive
+            url: >-
+              https://www.spice-space.org/download/releases/spice-protocol-0.14.4.tar.xz
+            sha256: 04ffba610d9fd441cfc47dfaa135d70096e60b1046d2119d8db2f8ea0d17d912
+      - name: phodav
+        buildsystem: meson
+        sources:
+          - type: archive
+            url: https://download.gnome.org/sources/phodav/3.0/phodav-3.0.tar.xz
+            sha256: 392ec2d06d50300dcff1ef269a2a985304e29bce3520002fca29f2edc1d138d1
+      - name: libcacard
+        sources:
+          - type: archive
+            url: >-
+              https://www.spice-space.org/download/libcacard/libcacard-2.8.1.tar.xz
+            sha256: fbbf4de8cb7db5bdff5ecb672ff0dbe6939fb9f344b900d51ba6295329a332e7
+      - name: python-pyparsing
+        buildsystem: simple
+        build-commands:
+          - pip3 install --prefix=/app pyparsing-2.4.6-py2.py3-none-any.whl
+        sources:
+          - type: file
+            url: >-
+              https://files.pythonhosted.org/packages/5d/bc/1e58593167fade7b544bfe9502a26dc860940a79ab306e651e7f13be68c2/pyparsing-2.4.6-py2.py3-none-any.whl
+            sha256: c342dccb5250c08d45fd6f8b4a559613ca603b57498511740e65cd11a2e7dcec
+  - shared-modules/libusb/libusb.json
+  - name: usbredir
+    buildsystem: meson
+    config-opts:
+      - "-Dtools=disabled"
+    sources:
+      - type: archive
+        url: https://spice-space.org/download/usbredir/usbredir-0.13.0.tar.xz
+        sha256: 4ba6faa02c0ae6deeb4c53883d66ab54b3a5899bead42ce4ded9568b9a7dc46e
```

Comment:
The build flags given to spice-gtk pass -Dusbredir=disabled - either that's wrong, or this module doesn't need to be built.

Reply:
I've got that from https://github.com/flathub/org.gnome.Boxes/blob/master/org.gnome.Boxes.json, but probably usbredir was for qemu...